infra/terraform/modules/lambda_mgmt/lambda.tf

```diff
@@ -0,0 +1,28 @@
+# Create the Lambda function with environment variables for filtering or config
+resource "aws_lambda_function" "lambda_function" {
+  function_name    = "${var.lambda_function_name}"
+  filename         = "${var.zipfile}"
+  handler          = "${var.handler}"
+  role             = "${aws_iam_role.lambda_role.arn}"
+  runtime          = "${var.lambda_runtime}"
+  source_code_hash = "${var.source_code_hash}"
+  count            = "${var.enabled}"
+  timeout          = "${var.timeout}"
+
+  environment {
+    variables {
+      "${var.env_key_1}" = "${var.env_value_1}"
```

Comment:
uh, meta ✨

Reply:
Finally what you said makes more sense. The reason why I wasn't "accepting" it was because for some reason I assumed that you couldn't control what was on the left of the `=` sign 🤷‍♂️ 

But this makes sense now, I mean, it's obviously a bit hacky but if it works it wouldn't change anything inside the lambda function 👍